packages/next/apollo.jsx

```diff
@@ -43,20 +46,46 @@ let createApolloClient = (initialState, ctx, apolloCacheConfig, apolloClientConf
         }
     }
 
+    const hasFiles = (data) => {
+        let i = 0
+        let found
+
+        const keys = Object.keys(data)
+
+        while (!found && i < keys.length) {
+            if (data[keys[i]] instanceof File || data[keys[i]] instanceof Blob) {
+                found = true
```

Comment:
You can do `break` here :)

Reply:
It will stop on the next iteration anyway)

Reply:
missed that line, sorry :)